12-offer-encoding.md

```diff
@@ -186,43 +186,46 @@ The human-readable prefix for offers is `lno`.
 
 1. `tlv_stream`: `offer`
 2. types:
-    1. type: 2 (`chains`)
+    1. type: 2 (`offer_chains`)
     2. data:
         * [`...*chain_hash`:`chains`]
-    1. type: 6 (`currency`)
+    1. type: 4 (`offer_metadata`)
+    2. data:
+        * [`...*byte`:`data`]
```

Comment:
This is not described below.

Reply:
Added "MAY set `offer_metadata` for its own use." just so it gets a mention, added Rationale.